unidist/core/backends/mpi/core/worker/task_store.py

```diff
@@ -38,6 +41,8 @@ def __init__(self):
         # Started async tasks
         self.background_tasks = set()
 
+        self.output_depends = weakref.WeakKeyDictionary()
```

Comment:
A comment should be added what this is for.

Reply:
Is it will be ok?